zigbee/Tutorials-and-Examples/Zigbee-Steup-Zigbee-Network-Without-Association.md

```diff
@@ -0,0 +1,237 @@
+
+
+# 1. Introduction
+This article will provide a guideline for how to setup Zigbee network without association process.
+
+# 2. Joining a Network Through Association
+As you probably know that setup a Zigbee network always required to establishing a new network on the Zigbee Coordinator side and permit device to join, and then the joining device should initiate the procedure for network discovery to find a joinable network. The beacon request will be sent out from the joining device, and router or coordinator will response the beacon request with beacon. 
```

Comment:
"setup a Zigbee network always required to establishing a new network"
--> setup a Zigbee network is always required to establish ...